app/views/layouts/error.html.erb

```diff
@@ -4,14 +4,19 @@
     <meta charset="utf-8">
     <title>OpenStreetMap</title>
     <%= stylesheet_link_tag "errors", :media => "screen" %>
-    <%= render :partial => "layouts/meta" %>
+    <%= render :partial => "layouts/head" %>
```

Comment:
`<head>` inside `<head>`?

You're proposing to load everything on 404 pages: jquery, leaflet, bootstrap, turbo etc. And you're doing this to set max width and add one link styled as a button, right?

Reply:
Well since you opened a new PR that "does this right" I will close this anyway.

Reply:
Looking [here](https://github.com/openstreetmap/openstreetmap-website/pull/1550#issuecomment-304481362) I can guess that the page was supposed to work when the asset pipeline is broken, although currently it includes a logo asset.